schemas/org.youtube/youtube/jsonschema/1-0-0

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a youtube player event",
+  "self": {
+    "vendor": "org.youtube",
+    "name": "youtube",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "avaliablePlaybackRates": {
+      "type": "array",
+      "description": "An array of playback rates in which the current video is available",
+      "items": {
+        "type": "number",
+        "minimum": 0,
+        "maximum": 9007199254740991
+      }
```

Comment:
Playback rates in previous schemas can have negative values too

Reply:
These are the rates the YouTube player provides in the dropdown menu on the UI. It actually looks like the range is currently 0.25 - 2 (https://youtube-eng.googleblog.com/2017/09/variable-speed-playback-on-mobile.html )(I can't find any newer source on the range, but I tried some videos and it still appears to be the case), so the minimum can probably be changed to `0.25`, and maximum reduced to `2`? (or a bit higher in case YouTube decides to allow higher values in future.

Reply:
I think leaving the minimum at 0 does no harm. The max could change if the Youtube API won't ever respond with a bigger number. Maybe going for something like 64 or 128 is a safe bet.

Reply:
The maximum has been changed to 128.